composer.json

```diff
@@ -13,7 +13,7 @@
         "irc": "irc://irc.freenode.net/phergie"
     },
     "require": {
-        "php": "^5.5||^7.0",
+        "php": "^5.6||^7.0",
```

Comment:
this is different then the rest of the plugins, I think at this point as we are moving away from 5.5, it would be good to make sure that we are standard in how we call for dependencies. 

@svpernova09 think it should be changed to match the rest of our composer.json's or leave it?